app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, source_file, new_file = command.split()

```

Comment:
You could prevent unpacking errors using *_ in case more arguments are passed
```suggestion
    command, source_file, new_file, *_ = command.split()

```